org.opent2t.sample.hub.superpopular/com.nest.hub/js/package.json

```diff
@@ -19,10 +19,12 @@
   },
   "homepage": "https://github.com/openT2T/translators#readme",
   "dependencies": {
-    "request": "^2.74.0",
-    "request-promise": "^4.1.1",
+    "firebase": "2.4.2",
     "opent2t": "^1.0.1",
-    "opent2t-onboarding-org-opent2t-onboarding-nest": "^0.0.1"
+    "opent2t-onboarding-org-opent2t-onboarding-nest": "^0.2.0",
```

Comment:
>opent2t-onboarding-org-opent2t-onboarding-nest": "^0.2.0" [](start = 5, length = 57)

Should be 0.0.1 which is the version on NPM and in package.json for the Nest onboarder in master.